src/inc/quic_var_int.h

```diff
@@ -65,13 +65,16 @@ QuicVarIntEncode(
         Buffer[0] = (uint8_t)Value;
         return Buffer + sizeof(uint8_t);
     } else if (Value < 0x4000) {
-        *(uint16_t*)Buffer = QuicByteSwapUint16((0x40 << 8) | (uint16_t)Value);
```

Comment:
I'm not a big fan of using temporary variables... I'm wondering if just writing out each individual byte assignment might be better.

Reply:
I checked for quant, and this gets optimized away completely.

Reply:
Ok.

Reply:
Other option would be to overwrite `Value`?